exporters/otlp/otlptrace/internal/otlptracetest/clienttest.go

```diff
@@ -0,0 +1,120 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Can we rename this file to `client.go`? It seems redundant to have `test` in both the package name and the file name.